pkgs/applications/kde/klettres.nix

```
@@ -3,7 +3,7 @@
 mkDerivation {
   pname = "klettres";
   meta = with lib; {
-    homepage = "https://kde.org/applications/en/utilities/org.kde.klettres";
+    homepage = "https://invent.kde.org/education/klettres";
```

https://github.com/NixOS/nixpkgs/pull/152490#discussion_r776116234

```suggestion
    homepage = "https://apps.kde.org/klettres/";
```